views/tx.tmpl

```diff
@@ -4,7 +4,16 @@
 {{template "html-head" printf "Decred Transaction %.20s..." .TxID}}
 <body>
 {{template "navbar"}}
-
+<script type="text/javascript">
+    ws.registerEvtHandler("newblock", function (event) {
+        console.log("tx Received newblock message", event);
+        var confirms = $(".confirmations")
+        if (isNaN(confirms.text())) {
+            confirms.text("0")
+        }
+        confirms.text(parseInt(confirms.text())+1)
```

Comment:
what's going on here?  I'm also confused why tx.tmpl has a `registerEvtHandler` running, but not address and block